tm2/pkg/sdk/params/doc.go

```diff
@@ -4,12 +4,55 @@
 // It includes a keeper for managing key-value pairs with module identifiers as
 // prefixes, along with a global querier for retrieving any key from any module.
 //
-// Changes: This version removes the concepts of subspaces and proposals,
-// allowing the creation of multiple keepers identified by a provided prefix.
-// Proposals may be added later when governance modules are introduced. The
-// transient store and .Modified helper have also been removed but can be
-// implemented later if needed. Keys are represented as strings instead of
-// []byte.
+// The Params Module provides functionalities for caching and persistent access
+// to parameters across the entire chain.
 //
+// It manages both module parameters and arbitrary parameters.
```

Comment:
This documentation wording needs to improve, and a lot of the information isn't applicable to tm2.
tm2/pkg/sdk/params has no knowledge of "realms". It just handles /xyz prefixes. 

documentation needs to split into two docs; the tm2 side, and the rest; the latter referencing the former for completeness.

Reply:
XXX